index.html

```diff
@@ -31,7 +33,7 @@ <h3 class="detail">I'm a software developer! I can help you build a product , fe
             <li><a href="#"> <img src="Vector.svg"class="socialmedia"/></a></li>
           </ul>
         </div>
-    </section>
+    </section></div>
     
```

Comment:
- [ ] KIndly make sure you use the correct path for your images. as you can see they are displaying 
![image](https://user-images.githubusercontent.com/97100091/193921940-3a948754-88ed-4cb7-ba1f-106204478e79.png)

`hint: src="./image/...`